bundles/framework/featuredata/view/FeatureDataContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tabs } from '../../../../src/react/components/Tabs';
+import styled from 'styled-components';
+
+const ContainerDiv = styled('div')`padding: 1em`;
```

Comment:
Tällaiset voisi rivittää vaikka enemmän rivejä vievätkin, mutta ovat helpompia lukea:
```suggestion
const ContainerDiv = styled('div')`
padding: 1em
`;
```